javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,15 @@
+const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+const today = new Date();
+const dayNumber = today.getDay();
+
+function getEventWeekday (daysToEvent){
+  const eventDayNumber = (dayNumber + daysToEvent) % 7;
+  return days[eventDayNumber];
+}
+
+console.log(getEventWeekday(5));
+console.log(getEventWeekday(9));
+
+//Here I used prompt for more practice :)
```

Comment:
Very nice. You can do that, although I think no one is ever using prompt in real life :)